packages/errors/src/react/error-page.tsx

```diff
@@ -37,28 +40,28 @@ const styles = {
 
     return media(
       '(min-width: 769px)',
-      stylesheet['@media (min-width: 769px)'][`.${label}`]
+      stylesheet['@media (min-width: 769px)']['.psds-error-page__text']
```

Comment:
If not using label, which I agree seems silly, should probably remove the label var.